lib/web/fetch/index.js

```diff
@@ -2165,9 +2178,9 @@ async function httpNetworkFetch (
             status,
             statusText,
             headersList,
-            body: decoders.length
-              ? pipeline(this.body, ...decoders, noop)
-              : this.body.on('error', noop)
+            body: streams === undefined
+              ? this.body.on('error', noop)
+              : pipeline(streams, noop).on('error', noop)
```

Comment:
this is just hiding a bug somewhere else

Reply:
how about now?